source/features/table-input.tsx

```diff
@@ -39,14 +39,19 @@ function init(): void {
 		anchor.after(
 			<details className="details-reset details-overlay flex-auto toolbar-item select-menu select-menu-modal-right hx_rsm">
 				<summary
-					className="text-center menu-target py-2 p-md-1 hx_rsm-trigger tooltipped tooltipped-n mx-1"
+					className="text-center menu-target py-2 p-md-1 hx_rsm-trigger ml-1"
```

Comment:
```suggestion
					className="text-center menu-target py-2 p-md-1 hx_rsm-trigger mx-1"
```

They all have `mx-1`